scapy/contrib/automotive/obd/scanner.py

```diff
@@ -0,0 +1,144 @@
+#! /usr/bin/env python
+# flake8: noqa: F405
+
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Andreas Korb <andreas.korb@e-mundo.de>
+# Copyright (C) Friedrich Feigel <friedrich.feigel@e-mundo.de>
+# This program is published under a GPLv2 license
+
+import sys
+from scapy.compat import chb
+from scapy.contrib.automotive.obd.obd import *
+
+
+def _supported_id_numbers(socket, timeout, service_class, id_name, verbose):
```

Comment:
could you please add some documentation for the parameters

Reply:
Done.

Reply:
Could you please use the same documentation style like other files. Here is an example: https://github.com/secdev/scapy/blob/a864193915adff0e00bb5a9bcd3f2ffd62f9e43d/scapy/contrib/isotp.py#L1830-L1854

Reply:
Oh sorry, of course!